src/garboWanderer.ts

```diff
@@ -0,0 +1,30 @@
+import { Effect, Location } from "kolmafia";
+import { globalOptions } from "./config";
+import { freeFightFamiliarData } from "./familiar/freeFightFamiliar";
+import { estimatedGarboTurns } from "./turns";
+import { WandererManager } from "./wanderer";
+import { $item, get } from "libram";
+import { garboValue } from "./garboValue";
+import { Potion } from "./potions";
+import { embezzlerCount } from "./embezzler";
+
+let _wanderer: WandererManager | undefined;
+export function wanderer(): WandererManager {
+  if (!_wanderer) {
+    _wanderer = new WandererManager({
+      ascend: globalOptions.ascend,
+      estimatedTurns: estimatedGarboTurns,
+      itemValue: garboValue,
+      effectValue: (effect: Effect, duration: number) =>
+        new Potion($item.none, { effect, duration }).gross(embezzlerCount()),
```

Comment:
I worry that `wanderer()` calling `embezzlerCount()` could cause problems. I don't think it will, but I'm sure you see my concern